app/src/main/java/ca/bc/gov/bchealth/ui/healthrecord/add/AddHealthRecordsFragment.kt

```diff
@@ -61,6 +61,13 @@ class AddHealthRecordsFragment : Fragment(R.layout.fragment_add_health_records)
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
+        if (args.isBackButtonEnabled) {
```

Comment:
As discussed adding text is not necessary. We can discuss this today and remove this.